src/IntegrationUtils.ts

```diff
@@ -322,6 +325,8 @@ function createNewIntegration(integrationFileUri: vscode.Uri, devMode? : boolean
 				console.error(error); 
 			});
 
+		devMode = extension.getDevMode();
```

Comment:
What is happening when use is using the "Start integration in dev mode" but never "started dev mode" or either "stopped dev mode" previously? it looks alke the command "Start integration in dev mode will not start in dev mode.

Reply:
Taking a look

Reply:
Updated